dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/JavaInfo.java

```diff
@@ -44,12 +44,12 @@ private JavaInfo() {
         MAJOR = Integer.valueOf(versionElements[i++]);
```

Comment:
Should these Integer.valueOf() calls also be updated?

Reply:
no, I intentionally only made non-critical version elements use the safe parsing mechanism. If we were to parse java major version as 0, we would break in all sorts of different places.